templates/Coaster/show.html.twig

```diff
@@ -357,6 +366,70 @@
 
       loadImage();
       loadReviews();
+
+      var options = {
+        series: [
+          {% for countRating in countRatings %}
+              {
+                name: {{ countRating.value }},
+                data: [{{ countRating.count }}]
+              },
+          {% endfor %}
+        ],
+        dataLabels: {
+          enabled: false
+        },
+        colors:['#bf4040', '#c24d31', '#c25c20', '#bd6b09', '#b57b00', '#a98a00', '#999800', '#84a600', '#6ab31f', '#40bf40'],
+        chart: {
+          type: 'bar',
+          toolbar: {
+            show: false
+          },
+          height: 100,
+          stacked: true,
+          stackType: '100%'
+        },
+        plotOptions: {
+          bar: {
+            horizontal: true,
+            barHeight: '100%'
+          },
+        },
+        stroke: {
+          width: 0
+        },
+        xaxis: {
+          categories: [''],
+          labels: {
+            formatter: function(value, timestamp, opts) {
+              return value + "%";
+            }
+          }
+        },
+        yaxis: {
+          labels: {
+            show: false
+          }
+        },
+        tooltip: {
+          y: {
+            formatter: function (val, { series, seriesIndex, dataPointIndex, w }) {
+              var value = Math.floor(w.globals.seriesPercent[seriesIndex][0] * 100) / 100;
```

Comment:
apparently you already have the raw value here: `series[seriesIndex]`

and dataPointIndex + w are no longer used in this case

Reply:
series contains raw value (i.e number of votes)
seriesIndex contains the index so it's an integer
w.globals.seriesPercent contains the percentage for each raw value in series

I remove dataPointIndex since I'm not using it